### PR TITLE
chore(examples): update local-tls example to use action configs

### LINKED ISSUE #3789: 0.13 example update: local-tls

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3789

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/local-tls/services/node-service/garden.yml [hunk 1]
```diff
@@ -1,24 +1,52 @@
...
+    - path: /hello
+      port: http
+    - path: /call-go-service
+      port: http
+
+---
+kind: Test
+name: node-service-unit
+description: Node service container unit tests
+type: exec
```
Comment (Walther) [inline examples/local-tls/services/node-service/garden.yml:31]: should these be `exec` or `container`?

File: examples/local-tls/services/node-service/garden.yml [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
+  - deploy.go-service
+
+spec:
+  command: [ npm, test ]
+
+---
+kind: Test
+name: node-service-integ
+description: Node service container integration tests
+type: exec
```
Comment (Walther) [inline examples/local-tls/services/node-service/garden.yml:44]: should these be `exec` or `container`?

**Review by Walther — APPROVED** [review]
Looks good!
Two tiny nits on the tests.

File: examples/local-tls/services/node-service/garden.yml (hunk 1 above)
Comment (edvald) [reply examples/local-tls/services/node-service/garden.yml:31]: Should be `container`.

Comment (vvagaytsev) [reply examples/local-tls/services/node-service/garden.yml]: Thanks! I fixed it in both tests.

File: examples/local-tls/services/node-service/garden.yml (hunk 2 above)
Comment (vvagaytsev) [reply examples/local-tls/services/node-service/garden.yml]: Changed to `container`.

**Pull request merged**